DotnetIntern2021/Basic/Exercise11.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DotnetIntern2021.Basic
+{
+    public class Exercise11
+    {
+        public int Reverse(int n)
+        {
+            String s = RadixChange(n);

```

Comment:
Thử tính toán ngay trên số được không ? đừng dùng xử lý chuỗi

Reply:
done